src/OrchardCore.Modules/OrchardCore.OpenId/Views/Application/Create.cshtml

```diff
@@ -165,114 +176,114 @@
 </form>
 <script at="Foot">
     //<![CDATA[
-    window.onload = function () {
+    window.onload = function() {
```

Comment:
Please undo these formatting changes.

Reply:
My bad, last time I'm using the 'Format Document' in VS.  This has been completed.